### ISSUE TITLE
NTR: Galactose-1-phosphate high in RBC (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8272]
New term request Galactose-1-phosphate high in RBC (elthat_101219010038): Gal-1P; erythrocytes, red blood cells

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000610
- label: Elevated erythrocyte galactose-1-phosphate concentration
- parents: [Abnormal erythrocyte metabolite concentration (HP:0034737)](https://hpo.jax.org/app/browse/term/HP:0034737)
- pmid: PMID:30172461
- def: Concentration of galactose-1-phosphate in red blood cells (erythrocytes) above the upper level of normal.
- comment: Impaired activity of galactose-1-phosphate uridyltransferase (GALT) causes classic galactosemia (OMIM 230400), characterized by the accumulation of galactose-1-phosphate (GAL1P) in patients' red blood cells (RBCs).
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8272
- synonyms:
